app/routes/settings+/profile.photo.tsx

```diff
@@ -21,17 +23,29 @@ import { authenticator, requireUserId } from '~/utils/auth.server'
 import { prisma } from '~/utils/db.server'
 import {
 	Button,
+	ErrorList,
 	getFieldsFromSchema,
 	LabelButton,
-	preprocessFormData,
-	useForm,
 } from '~/utils/forms'
 import { getUserImgSrc } from '~/utils/misc'
 
 const MAX_SIZE = 1024 * 1024 * 3 // 3MB
```

Comment:
3 MB could be quite restrictive as a default. 5 MB may be?

Reply:
From my side gladly, but I think the whole thing will be interesting later when the whole file and image strategy solidifies. From what I've seen in videos (especially the interview with Kurt Mackey) the plan could be to outsource the whole image strategy, maybe a separate SQLite just for images, that depends on how small or large you want to keep the images and DB in general.

Reply:
I'm thinking it would be great to figure out a way to compress images before upload. I'm pretty sure that's possible.